presto-common/src/main/java/com/facebook/presto/common/zorder/ZOrder.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.common.zorder;
```

Comment:
Let's not put it in presto-common, presto-hive-common is a better place.

Reply:
presto-hive-common does not seem to have a "test" folder like presto-common; in this case, where would you recommend I move the TestZOrderIntegers class?

Reply:
I think we can create the folder accordingly. Please try to see if it works.